spec/components/text_sentinel_spec.rb

```diff
@@ -74,6 +74,11 @@
       expect(TextSentinel.new(valid_string.upcase)).not_to be_valid
     end
 
+    it "allows all caps topics when loud posts are allowed" do
+      SiteSetting.stubs(:allow_uppercase_posts).returns(true)
```

Comment:
Sure if every other test does it that way, I'm okay with it. But we prefer not to mock in the future. At some point someone should fix this file.